### ISSUE TITLE
Error Installing plugin

### ISSUE THREAD

**Issue by mnadeem** [de-jcup/eclipse-jenkins-editor#75]
An error occurred while collecting items to be installed
session context was:(profile=C__Users_mnadeem6_softwares_eclipse_jee-2020-03_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Unable to read repository at https://dl.bintray.com/de-jcup/jenkinseditor/plugins/de.jcup.jenkinseditor.plugin_1.7.0.jar.
sun.security.validator.ValidatorException: TLS Server certificate issued after 2019-04-16 and anchored by a distrusted legacy Symantec root CA: CN=VeriSign Class 3 Public Primary Certification Authority - G5, OU="(c) 2006 VeriSign, Inc. - For authorized use only", OU=VeriSign Trust Network, O="VeriSign, Inc.", C=US

**Comment by mnadeem** [comment]
I believe the problem is with JRE and not with JDK

**Comment by de-jcup** [comment]
I think it's a problem about the certificate store of your used java version  - I just added an entry into [Wiki-FAQ](https://github.com/de-jcup/eclipse-jenkins-editor/wiki/FAQ#i-am-not-able-to-install-plugin-by-marketplace) and described situation and possible solution. 

tl;dr: I would suggest to install a new Java Version with an up-to-date certificate store.

**Comment by de-jcup** [comment]
Please try again, but do the update by eclipse marketplace client, read #90 for more details
